### DISCUSSION TITLE
能不能添加无用前置mod的选项

### DISCUSSION THREAD

**Discussion by hyk090307** [Meloong-Git/PCL#828] [提问] [answered]
安装某些mod会有前置mod，删除mod后前置mod还在，很占空间，拖慢启动速度

**Comment by hyk090307** [comment]
同时如果前置库无用禁用mod也禁用前置mod

**Comment by WForst-Breeze** [comment, marked as answer]
可能不会制作，请见 #179 ，出于技术上的原因，很难识别一个 mod 是否为另一个 mod 的前置 mod，还有用户提议让使用者手动标注，也同样被拒绝，请见 #816 #818
